### ISSUE TITLE
Yii2 support

### ISSUE THREAD

**Issue by smartycoder** [dunglas/frankenphp#604]
### Describe you feature request

Is there a plan to support Yii2 framework or is there any example for docker.

**Comment by razvanphp** [comment]
It just works, you only need to link `web` folder to `public` or change the default `Caddyfile`.

**Comment by dunglas** [comment]
Would you mind opening a docs PR to explain how to serve Yii?

**Comment by razvanphp** [comment]
Yes, sure, I'm currently building a PR for [yiisoft/yii2-docker](https://github.com/yiisoft/yii2-docker).

The official Yii2 images also inherit from official PHP docker, but I just miss a way to extrapolate from something like
`PHP_BASE_IMAGE_VERSION=8.2-zts` into the tag for frankenphp docker image. Ideally it should be the exact same PHP version, down to path version, but it can be that they MINOR version might be out of sync and symbols might not be found.

Any suggestion on how to relink the frankenphp binary to the actual php installed? Or the compilation from source is the only way?

Current approach:
```
COPY --from=dunglas/frankenphp /usr/local/bin/frankenphp /usr/local/bin/frankenphp
```

**Comment by bahirul** [comment]
I think we need the worker script for yii2, with classic no such big diff compare to apache + fpm

**Comment by terabytesoftw** [comment]
Library for worker mode https://github.com/yii2-extensions/franken-php

**Comment by samdark** [comment]
This one is done and could be closed unless one plans to add all the adapter links to docs.

**Comment by henderkes** [comment]
Thanks

**Comment by dunglas** [comment]
Mentioning the adapter in the docs could be nice!

**Comment by terabytesoftw** [comment]
> Mentioning the adapting in the docs could be nice!

Should i do a PR?

**Comment by dunglas** [comment]
Yes please!
